### PR TITLE
Use the SwiftPM overrides consistently

### PR DESCRIPTION
**Pull request by hartbit**
I need this PR in to help me work on `swift-integration-tests` more efficiently

### CODE REVIEW

**Comment by hartbit** [comment]
@aciidb0mb3r does that look reasonable to you?

File: example-package-dealer.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # RUN: git clone https://github.com/apple/example-package-dealer %t.dir/dealer
 
 # RUN: rm -rf %t.dir/dealer/.build
-# RUN: %{swift} build --package-path %t.dir/dealer 2>&1 | tee %t.build-log
+# RUN: %{swift-build} --package-path %t.dir/dealer 2>&1 | tee %t.build-log
```
Comment (aciidb0mb3r) [inline example-package-dealer.py]: I don't think we should do this because then we're not testing swift's subtool invocation.

Comment (hartbit) [reply example-package-dealer.py]: We're testing the swift's subtool invocation by default, when not passing `swiftpm_build`.

**Comment by hartbit** [comment]
Any news?

File: example-package-dealer.py (hunk 1 above)
Comment (aciidb0mb3r) [reply example-package-dealer.py]: Ah okay. This repo doesn't have PR testing so maybe create a dummy PR on swift and then do a cross-repo testing?

**Comment by hartbit** [comment]
Got tests passing on Swift with a dummy PR

**Comment by aciidb0mb3r** [comment]
Thanks!

**Pull request merged**